tests/UI/screenshot-diffs/missing-expected.list

```diff
@@ -0,0 +1,28 @@
+Cohorts_metric_select_opened.png
```

Comment:
This file should be removed again...

Reply:
Thank you for catching that @sgiehl . I guess that's what I get for using `git add tests/UI` instead of being more specific.